### DISCUSSION TITLE
problem php 8.2

### DISCUSSION THREAD

**Discussion by tecnoweb** [Laravel-Backpack/community-forum#820] [Q&A (Help)]
I have installed backpack dev tools after install I have more deprecated function and this error Doctrine\DBAL\Exception\SyntaxErrorException
An exception occurred while executing a query: SQLSTATE[HY000]: General error: 1 near "(": syntax error

Return type of Backpack\DevTools\CustomFile::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in

is not supported by php 8.2 ?

**Comment by karandatwani92** [comment]
Hey @tecnoweb 
can you provide the output of ```php artisan backpack:version```. It will be helpful to identify & resolve your issue.

Thanks

**Comment by tecnoweb** [comment]
Hello @karandatwani92 

### LARAVEL VERSION:
v8.83.27@e1afe088b4ca613fb96dc57e6d8dbcb8cc2c6b49

### BACKPACK PACKAGE VERSIONS:
backpack/crud: 5.6.1
backpack/devtools: 1.3.5
backpack/generators: v3.3.17
backpack/pro: 1.6.6

**Comment by karandatwani92** [comment]
Hey @tecnoweb 

Sorry, these are very old versions. I can't help here.

Can you get the latest version of them?

Thanks

**Reply by tecnoweb** [reply]
but I have installed today from documentation  I don't know if is good information to you I will use plesk

**Reply by karandatwani92** [reply]
I just noticed, you are using Laravel 8.
Backpack v6 requires Laravel 10.
**Requirements**
https://backpackforlaravel.com/docs/6.x/installation#requirements-1
